### PR TITLE
Home Work 8

### PR DESCRIPTION
**Pull request by TanyaB777**

### CODE REVIEW

File: src/main/java/hw_8/SessionCloser.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package hw_8;
+
+import java.util.List;
+
+public interface SessionCloser {
```
Comment (AntBelyi) [inline src/main/java/hw_8/SessionCloser.java:6]: Add @FunctionalInterface annotation and read about that. If I'm not mistaken we considered this annotation on our previous lection

File: src/main/java/hw_8/Program.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+public class Program {
+    public static void main(String[] args) {
+
+        MyBrowser browser = new MyBrowser();
+        MyFile file = new MyFile();
+
+        Lecture30.finishWork(browser);
+        Lecture30.finishWork(file);
+
+        SessionCloser sessionCloser = (entities) -> {
```
Comment (AntBelyi) [inline src/main/java/hw_8/Program.java]: "(" and ")" can be removed if only one argument is used

Comment (TanyaB777) [reply src/main/java/hw_8/Program.java]: fixed

File: src/main/java/hw_8/SessionCloser.java (hunk 1 above)
Comment (TanyaB777) [reply src/main/java/hw_8/SessionCloser.java:6]: fixed

**Pull request merged**